pkg/cephfs/util.go

```diff
@@ -30,61 +30,41 @@ import (
 
 	"github.com/ceph/ceph-csi/pkg/util"
 	"github.com/container-storage-interface/spec/lib/go/csi"
+	"k8s.io/kubernetes/pkg/util/keymutex"
 	"k8s.io/kubernetes/pkg/util/mount"
 )
 
 type volumeID string
 
-func makeVolumeID(volName string) volumeID {
-	return volumeID("csi-cephfs-" + volName)
+func mustUnlock(m keymutex.KeyMutex, key string) {
+	if err := m.UnlockKey(key); err != nil {
+		klog.Fatalf("failed to unlock mutex for %s: %v", key, err)
```

Comment:
`klog.Fatalf` calls `os.Exit(255)` do we need this? as the lock uses the in-memory cache  this can cause an issue I think,

Reply:
Failing to unlock a mutex is such a critical error that imho the program should abort. The requests would just keep piling up anyway and they would never finish because the mutex is in unknown state (maybe still locked for whatever reason)